openeo/config.py

```diff
@@ -180,7 +180,16 @@ def get_config() -> ClientConfig:
     global _global_config
     if _global_config is None:
         _global_config = ConfigLoader.load()
-        message = f"Loaded openEO client config from sources {_global_config.sources}"
+
+        # In this case we avoid f-strings and use a more complicated expression,
+        # because on Windows we already have '\\' in the paths and then the f-string
+        # will escape those a *second* time, so we end up with '\\\\' .
+        # That is unreadable, and also a lot harder to deal with in unit tests.
```

Comment:
I think this explanation is a bit misleading as escaping and double escaping issue is because of implicit `repr` in the pytest reporting, it's not related to f-strings or especially relevant in this non-test code path.

e.g. in #396  I'm trying a variant that still uses an f-string to keep things a bit more compact